core/dbt/tests/fixtures/project.py

```diff
@@ -12,6 +15,8 @@
 
 # These are the fixtures that are used in dbt core functional tests
 
+INITIAL_ROOT = os.getcwd()
```

Comment:
There's a builtin pytest fixture thing that you can use instead of this INITIAL_ROOT variable: request.config.rootdir.  If you add the 'request' fixture to the signature of the 'logs_dir' fixture, that should work.